proposals/NNNN-rc-sdl-module.md

```diff
@@ -0,0 +1,239 @@
+# Remote Control - SDL Module
+
+* Proposal: [SDL-nnnn](NNNN-rc-sdl-module.md)
+* Author: [Joey Grover](https://github.com/joeygrover/)
+* Status: **Awaiting Review**
+* Impacted Platforms: [RPC]
+
+## Introduction
+
+There is a strong desire to implement some method of inter-app communication between SDL apps. The SDL protocol can be enhanced to facilitate this feature. Leveraging the previously accepted Remote Control functionality, we can introduce a new module type, SDL_APP.
+
+
+## Motivation
+
+SDL connected apps provide augmented functionality to an SDL enabled IVI system. This includes mobile navigation or even off-board voice recognition. Most apps focus on a particular feature or experience they provide and therefore leverage libraries, 3rd party solutions, or other apps to perform the features that not part of their main line user experience. One of the ways an app augments itself is using SDL enabled features, send location, voice req, etc.
+
+
+## Proposed solution
+
+This proposal introduces a new module type into the Remote Control feature titled SdlApp.
+
+Core would monitor which apps are connected and create the virtual modules for each of the SDL apps that are connected and have registered as a RC module. When an app connects, it can register with a flag that will allow other apps to access/subscribe to that app. Core takes this flag and creates a new virtual module, and sends out an `OnRCStatus` notification that includes this new module.   
+
+#### SDL App Identification
+In order to talk to specific modules we will need to introduce some sort of identification. For this reason we will add a `moduleUUID` field to the `GetInteriorVehicleData` request and m`ModuleData`.
+
+#### Subscription
+When an app is subscribed to another app,(App A is subscribed to App B), Core will take the RPC sent by App B and create `OnInteriorVehicleData` RPCs for the corresponding SDL_APP module created for App B. It should then send an update to all subscribed apps, in this case App A. 
+
+#### SDL App becomes available
+When a new app registers with an available RC service, the Core module will be responsible for sending `OnRCStatus` that contains the new virtualized module.
+
+#### Use Case
+Apps can display `Show` updates using the supplied metadata and HMI type to determine where it should be displayed. An example of when this is useful is for a passenger app that allows the user to see what is playing through the SDL connected app as well as have basic control.
```

Comment:
I think the first sentence should either read "Apps can display...where they should be displayed." or "An app can display...where it should be displayed."